docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -194,81 +193,43 @@ This page details the specifics around testing Ansible Networking modules.
    Integration tests, although not required, are a welcome addition.
    How to do this is explained in the rest of this document.
```

Comment:
This line should probably change now that the "rest of this document" is only two lines long. I'd merge lines 193 through 196: "Integration tests, although not required, are a welcome addition. To run a network integration test, issue a command in the form:"

Reply:
Duh. My eye was fooled by the long swath of red and missed the rest of the doc. I need more coffee?